src/grid/GridContainer.test.tsx

```diff
@@ -105,27 +105,6 @@ describe('GridContainer', () => {
     expect(mockVotesDbChildSet).not.toBeCalled();
   });
 
-  it('doesnt allow voting when already voted', () => {
```

Comment:
Could keep the test but reverse the condition. i.e. "allows voting when already voted" ?

Reply:
Good idea, done!